overlord/ifacestate/ifacemgr.go

```diff
@@ -122,3 +126,54 @@ func MockSecurityBackends(be []interfaces.SecurityBackend) func() {
 	backends.All = be
 	return func() { backends.All = old }
 }
+
+// SnapInterfaceState describes state of interfaces of a single snap.
+type SnapInterfaceState struct {
+	// Revision is the revision of the that snap was loaded into the
+	// interface repository. When snapd restarts it may have added a snap to
+	// the interface repository even when that snap was not yet active. To
+	// ensure that we can reload the interface repository correctly, for
+	// inactive snaps, this is the revision to load on startup.
+	Revision snap.Revision `json:"revision"`
+}
+
+// ifaceRepoKey is the key used to store state of the interface repository in the snapd state.
+const ifaceRepoKey = "repo"
+
+// Get retrieves the SnapInterfaceState of the given snap or ErrNoState if missing.
+func Get(st *state.State, snapName string, snapifst *SnapInterfaceState) error {
```

Comment:
Get and Set names for these methods sound a little too generic (although we have similiar case in snapstate, so perhaps someone else can wiegh on that). Wheter these names stay or change, I think it would be nice to have them in a new file under ifacestate.

Reply:
I can move them to `helpers.go`, would that be okay?

Reply:
Moved